lvm-driver.yaml

```diff
@@ -0,0 +1,48 @@
+package:
+  name: lvm-driver
+  version: 1.6.2
+  epoch: 0
+  description: Dynamically provision Stateful Persistent Node-Local Volumes & Filesystems for Kubernetes that is integrated with a backend LVM2 data storage stack.
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    runtime:
+      - btrfs-progs
+      - busybox
+      - ca-certificates-bundle
+      - device-mapper
+      - e2fsprogs
+      - e2fsprogs-extra
+      - lvm2
+      - util-linux
+      - xfsprogs
```

Comment:
upstream runtime deps are defined here. 
https://github.com/openebs/lvm-localpv/blob/45e0fbdd6af38f3025132fff5b6d10c3c2eec1fb/buildscripts/lvm-driver/Dockerfile#L15-L18

there are few deviations because of package naming and availability in wolfi.
- lvm2-extra and xfsprogs-extra were not available so I added the dev variant which were present in wolfi. 
```bash
# in alpine lvm2-extra provides following 
/ # apk info -L lvm2-extra
lvm2-extra-2.03.23-r3 contains:
sbin/blkdeactivate
sbin/fsadm
sbin/lvm_import_vdo
sbin/lvmdump
# in wolfi all of them are part of lvm2 package itself.
/ # apk search cmd:fsadm
lvm2-2.03.26-r0
/ # apk search cmd:blkdeactivate
lvm2-2.03.26-r0
/ # apk search cmd:lvmdump
lvm2-2.03.26-r0
/ # apk search cmd:lvm_import_vdo
lvm2-2.03.26-r0
/ #
```
- same goes for `xfsprogs` package. 
- libc6-compat is not available in wolfi.
```bash
$ docker run --rm -it alpine sh
/ # apk add libc6-compat
fetch https://dl-cdn.alpinelinux.org/alpine/v3.20/main/x86_64/APKINDEX.tar.gz
fetch https://dl-cdn.alpinelinux.org/alpine/v3.20/community/x86_64/APKINDEX.tar.gz
(1/3) Installing musl-obstack (1.2.3-r2)
(2/3) Installing libucontext (1.2-r3)
(3/3) Installing gcompat (1.1.0-r4)
OK: 8 MiB in 17 packages
/ # apk info -L libc6-compat
gcompat-1.1.0-r4 contains:
lib/ld-linux-x86-64.so.2
lib/libc.so.6
lib/libcrypt.so.1
lib/libgcompat.so.0
lib/libm.so.6
lib/libpthread.so.0
lib/libresolv.so.2
lib/librt.so.1
lib/libutil.so.1
lib64/ld-linux-x86-64.so.2
```

Reply:
Looking at https://pkgs.alpinelinux.org/package/v3.18/main/x86_64/libc6-compat I'm not sure if a musl compatibility is required here.